docs/source/platform_support.rst

```diff
@@ -66,16 +67,15 @@ This version of Python can be used with cocotb unless noted otherwise.
 Supported Windows Versions
 ==========================
 
-* **Windows 10 x86_64**
```

Comment:
It's supported until the end of this year.

Reply:
Added it back.